src/core/Core.js

```diff
@@ -87,7 +87,6 @@ class Uppy {
     this.upload = this.upload.bind(this)
 
     this.emitter = ee()
-    this.on = this.emitter.on.bind(this.emitter)
```

Comment:
`on` likely still needs a bind, similar to `this.resetProgress = this.resetProgress.bind(this)` correct?

Reply:
not strictly necessary now because `uppy.on()` is now an instance method of `uppy` and not of `uppy.emitter.on`, but probably good for consistency with the `emit` and `off` methods yes